src/profile/forms/Country.jsx

```diff
@@ -84,7 +84,7 @@ class Country extends React.Component {
                     value={country}
                     onChange={this.handleChange}
                   >
-                    <option value="" />
+                    <option value="">&nbsp;</option>
```

Comment:
Intended?

Reply:
Yeah - the linter didn't like it having some textual content.